.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: "Close stale issues"
+
+# Controls when the workflow will run. 
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  # Cron job time seems to be UTC  
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # stale:
+  stale-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: "This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. If you need additional assistance please contact Esri Technical Support. Thank you for your contributions."
+          # stale-pr-message: "This PR has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions."
+          close-issue-message: "This issue has been automatically closed due to inactivity. If you need additional assistance please contact Esri Technical Support."
+          # close-pr-message: "This PR has been automatically closed due to inactivity."
+          # exempt-issue-labels: "0 - new,1 - assigned"
+          only-labels: null
+          days-before-issue-stale: 365          
```

Comment:
Maybe we should remove the `issue` in these two, and then remove the commented out two matching ...-pr-... ones.  Thus treating both issues and PR's the same. 
Maybe... :)

Reply:
Ah, interesting. Let's give a try. I just pushed another commit.

Reply:
@bsvensson with the latest commit we now have messages for stale issues and stale PRs.